feature-set/src/lib.rs

```diff
@@ -1360,7 +1362,6 @@ pub static FEATURE_NAMES: LazyLock<AHashMap<Pubkey, &'static str>> = LazyLock::n
         (enshrine_slashing_program::id(), "SIMD-0204: Slashable event verification"),
         (enable_extend_program_checked::id(), "Enable ExtendProgramChecked instruction"),
         (formalize_loaded_transaction_data_size::id(), "SIMD-0186: Loaded transaction data size specification"),
-        (alpenglow::id(), "Enable Alpenglow"),
```

Comment:
should we maybe leave this line but comment it out to make it even more explicit what the intention is?

Reply:
done

Reply:
The SIMD that will go through governance will be https://github.com/solana-foundation/solana-improvement-documents/pull/326. I can add the number when we uncomment the feature flag.

Reply:
I'll go along w/ whatever you guys think. Just paranoid someone will think the feature is missing on accident, not see the comment, and try to add it in